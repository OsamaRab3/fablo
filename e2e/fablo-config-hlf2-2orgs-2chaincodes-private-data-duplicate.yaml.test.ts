import performTests from "./performTests";

const config = "samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml";

describe(config, () => {
  performTests(config);
});
