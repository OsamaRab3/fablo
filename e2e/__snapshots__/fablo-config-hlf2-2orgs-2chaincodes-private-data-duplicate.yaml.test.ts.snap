// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml should create proper e2e/__tmp__/samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml.tmpdir/fablo-target/.config/configstore/update-notifier-yo.json from samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml 1`] = `
"{
	"optOut": false,
	"lastUpdateCheck": 1755290146858
}"
`;

exports[`samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml should create proper files from samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml 1`] = `
[
  "e2e/__tmp__/samples/fablo-config-hlf2-2orgs-2chaincodes-private-data-duplicate.yaml.tmpdir/fablo-target/.config/configstore/update-notifier-yo.json",
]
`;
